env/SomaticFusion.yml

```diff
@@ -1,23 +1,42 @@
 name: SomaticFusion
```

Comment:
I can't get the new conda env to build on the mac, I think it's got too many system dependencies, it always does that when you export an env. Can you try removing some of the weird ones an leave just the ones you want to add, then see if it will build? I can explain if I'm not making sense

Reply:
I have removed the endings from the yml file (where they had more than one equals sign"). It still won't build on the mac, but it now builds correctly on linux